icons/git-pull-request-2.svg

```diff
@@ -0,0 +1,17 @@
+<svg
```

Comment:
I'm not pleased with the current size of the arrow from this one. 

I tried to give one a shot. But It's hard. I came up with two alternatives, but they have some tradeoffs..

Reply:
Forgot to post the screenshot. Let me know what you think
<img width="809" alt="Screenshot 2023-04-10 at 11 28 12" src="https://user-images.githubusercontent.com/11825403/231103062-970affa4-8a86-4271-946f-f7d64fd89415.png">

Reply:
@ericfennis I think alt 1…

Reply:
What do you think @karsa-mistmere ?

Reply:
Honestly, I guess I'm mostly in favour of keeping just the ones with no arrows. 🤷‍♂️

To me, both alternatives feel forced; Alt 1 will make our git icons inconsistent and Alt 2 doesn't quite stick to the guidelines. 🙈

Reply:
> Alt 1 will make our git icons inconsistent

Although they could easily be made consistent… Also visually centred and proportionally more square:

<img width="418" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/233218329-33c3b0b1-eaea-486c-92ba-a8c5d2bbb839.png">